jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/bucketEditor/S3KeyNode.kt

```diff
@@ -59,6 +62,15 @@ class S3KeyNode(val virtualFile: VirtualFile) : SimpleNode() {
         }
     }
 
+    fun resetLimitsForSearch(){
+        updateLimitsOnSizeChange()
+        prev = MIN_SIZE
+        next = currSize
+    }
+
+    /**
```

Comment:
nit: consts should be self-descriptive and the doc should be on the val itself, if necessary.